runHarness/AppInstance/AppInstance.pm

```diff
@@ -707,15 +711,23 @@ sub isRunningAndUpServices {
 	if ( !$allIsRunning ) {
 		$console_logger->error(
 			"Couldn't bring to running all $serviceTier services for appInstance $appInstanceNum of workload $workloadNum." );
-		exit 1;
+		if ($forked) {
+			exit;
```

Comment:
With the current implementation of callBooleanMethod..., the use of exit rather than exit 1 is correct.  However we will need to change these to exit 1 once we have fixed callBoolean... However, I do not want to hold this pull request up for those changes.